src/components/reusable/checkbox/checkboxGroup.ts

```diff
@@ -150,7 +151,9 @@ export class CheckboxGroup extends FormMixin(LitElement) {
               : null}
             <slot name="label"></slot>
           </legend>
-
+          <div class="description-text">
+            <slot name="description"></slot>
```

Comment:
Is there a reason/use case that this needs to be a slot vs a string property?

Reply:
There may be instances where certain portions of the text need to be bold or emphasized. In such cases, developers can provide an HTML template in the description Text.